### DISCUSSION TITLE
Please specify a platform for this target in your Podfile

### DISCUSSION THREAD

**Discussion by felixgodo** [codemagic-ci-cd/codemagic-docs#2714] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Codemagic error build Runner: 
Error output from CocoaPods:
↳

    [!] Automatically assigning platform `iOS` with version `12.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

Error: The plugin "google_maps_flutter_ios" requires a higher minimum iOS deployment version than your application is targeting.
To build, increase your application's deployment target to at least 14.0 as described at https://flutter.dev/to/ios-deploy
Error running pod install

I have tried to add the podfile myself as described in this discussion "https://github.com/orgs/codemagic-ci-cd/discussions/1917", but unfortunately it does not work...

### Expected results

a successful build...

### Actual results

Error output from CocoaPods:
↳

    [!] Automatically assigning platform `iOS` with version `12.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

Error: The plugin "google_maps_flutter_ios" requires a higher minimum iOS deployment version than your application is targeting.
To build, increase your application's deployment target to at least 14.0 as described at https://flutter.dev/to/ios-deploy
Error running pod install

### Build id (optional)

66d854413935f1780fcb22f1

**Comment by himesh-cm** [comment, marked as answer]
Hey @kingingo 

The error you are encountering indicates that the `google_maps_flutter_ios` plugin requires a higher iOS deployment target than the one currently set for your Flutter project.

To resolve this issue, follow the steps below:

1. Open the `ios/Podfile` file in a text editor.
2. Find the line that specifies the platform version and update it to 14.0.

```
platform :ios, '14.0'
```

If the above is not working for you, please share your `podfile` for further debugging.
